### PR TITLE
Add Frontend for feedbacks page

### PR DESCRIPTION
**Pull request by Hiw**

### CODE REVIEW

**Comment by Hiw** [comment]
[React Issue 1: Add Footer](https://trello.com/c/gqUwoYFz/361-react-issue-1-add-footer)

File: app/javascript/image_sharing/test/components/App.spec.js [hunk 1]
```diff
@@ -10,4 +12,61 @@ describe('<App />', () => {
...
+    nameForm.simulate('change', event);
+
+    expect(wrapper.state('name')).to.equal(name);
+  });
+
+  it('should have comments field', () => {
+    const wrapper = mount(<App />);
+    const commentsForm = wrapper.find('#commentsForm').hostNodes();
+    expect(commentsForm).to.have.length(1);
+  });
```
Comment (eric199wen) [inline app/javascript/image_sharing/test/components/App.spec.js:51]: I often consolidate these tests into one test named `it should render correctly` lol

File: config/routes.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  root 'application#home'
+  root 'feedbacks#new'
```
Comment (eric199wen) [inline config/routes.rb:3]: Is it necessary to change the root?

File: .circleci/config.yml [hunk 3]
```diff
@@ -22,7 +22,7 @@ jobs:
       # Install cmake, build-eseential since gem build with native extension needs it
       - run:
           name: Install cmake needed by rugged
-          command: sudo apt-get install cmake
+          command: sudo apt-get update && sudo apt-get install cmake
```
Comment (eric199wen) [inline .circleci/config.yml:25]: Maybe you can submit a PR to the original repo after ropes!

**Review by eric199wen — APPROVED** [review]

**Pull request merged**